### PR TITLE
Background image adjustments for KvContentfulImg

### PR DESCRIPTION
**Pull request by emuvente**

### CODE REVIEW

File: @kiva/kv-components/vue/KvContentfulImg.vue [hunk 1]
```diff
@@ -86,7 +91,6 @@ export default {
 		* */
 		fallbackFormat: {
 			type: String,
-			required: true,
```
Comment (emuvente) [inline @kiva/kv-components/vue/KvContentfulImg.vue:89]: Since this prop has a default it doesn't need to be required

Comment (mcstover) [reply @kiva/kv-components/vue/KvContentfulImg.vue:89]: Is that field needed to activate the validator?

File: @kiva/kv-components/vue/KvContentfulImg.vue [hunk 2]
```diff
@@ -45,6 +45,11 @@
 			<!-- browser doesn't support webp or browser doesn't support picture element -->
 			<img
 				class="tw-max-w-full tw-max-h-full"
+				style="
+					width: inherit;
+					height: inherit;
+					object-fit: inherit;
```
Comment (mcstover) [inline @kiva/kv-components/vue/KvContentfulImg.vue:51]: guessing we don't have tw classes for these.

File: @kiva/kv-components/vue/KvContentfulImg.vue (hunk 1 above)
Comment (mcstover) [reply @kiva/kv-components/vue/KvContentfulImg.vue:89]: no it doesn't appear to be.

**Review by mcstover — APPROVED** [review]

File: @kiva/kv-tokens/configs/tailwind.config.js [hunk 3]
```diff
@@ -124,6 +124,10 @@ module.exports = {
 		}),
 		extend: {
 			typography: kivaTypography.proseOverrides, // prose plugin overrides
+			height: {
+				40: rem(320),
```
Comment (ryan-ludwig) [inline @kiva/kv-tokens/configs/tailwind.config.js:128]: I think this works well for now. If we end up pulling in the aspect-ratio plugin later on, it'd be nice to use that and keep these misc. height values out of the config.

**Review by ryan-ludwig — APPROVED** [review]

File: @kiva/kv-components/vue/KvContentfulImg.vue (hunk 2 above)
Comment (emuvente) [reply @kiva/kv-components/vue/KvContentfulImg.vue:51]: None that I could find

**Pull request merged**